.env.local

```
@@ -1,4 +1,4 @@
-GOOGLE_GENAI_API_KEY=AIzaSyAnCOy-y59rYxIqvzTtHf3GR979sbwcXaY
 NEXT_PUBLIC_SUPABASE_URL=https://llsifflkfjogjagmbmpi.supabase.co
 NEXT_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Imxsc2lmZmxrZmpvZ2phZ21ibXBpIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NDU1MzY5OTMsImV4cCI6MjA2MTExMjk5M30.4uOd3OY7lhe1kYjROF5Y_2ob1v8Nm-QIMFx6MOzq1Vs
-SUPABASE_SERVICE_ROLE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Imxsc2lmZmxrZmpvZ2phZ21ibXBpIiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImlhdCI6MTc0NTUzNjk5MywiZXhwIjoyMDYxMTEyOTkzfQ.ZhKjCTF2f1sN8T5jKlAJdwH-3nT4sdaX7tODYeeIx74
\ No newline at end of file
+UPLOADTHING_TOKEN='eyJhcGlLZXkiOiJza19saXzlXzZjNTEzMTNhODBjOWJhZjc3ZTtk4NGNjYWZkNWI1YTEwYjE3MWZhODM1MTE5MzdhZjEyNGYyMjAwZzE2MGQi'
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)